### PR TITLE
[improvement] reduce memory footprint of column_reader and segment

### PR DESCRIPTION
**Pull request by dataroaring**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/rowset/segment_v2/bitmap_index_reader.cpp [hunk 1]
```diff
@@ -32,16 +32,15 @@
 namespace doris {
 namespace segment_v2 {
 
-Status BitmapIndexReader::load(bool use_page_cache, bool kept_in_memory,
-                               const BitmapIndexPB* index_meta) {
+Status BitmapIndexReader::load(bool use_page_cache, bool kept_in_memory) {
```
Comment (airborne12) [inline be/src/olap/rowset/segment_v2/bitmap_index_reader.cpp:35]: need to change UT

**Comment by dataroaring** [comment]
run buildall

**Review by Yukang-Lian — APPROVED** [review]
LGTM

**Review by platoneko — APPROVED** [review]

**Comment by dataroaring** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**